mkosi/distributions/rhel_ubi.py

```diff
@@ -0,0 +1,60 @@
+# SPDX-License-Identifier: LGPL-2.1+
+
+import urllib.parse
+from typing import Generator
+
+from mkosi.config import MkosiConfig
+from mkosi.distributions import centos
+from mkosi.installer.dnf import Repo
+
+
+def join_mirror(config: MkosiConfig, link: str) -> str:
+    assert config.mirror is not None
+    return urllib.parse.urljoin(config.mirror, link)
+
+
+class Installer(centos.Installer):
+    @staticmethod
+    def gpgurls() -> tuple[str, ...]:
+        return (
+            "https://access.redhat.com/security/data/fd431d51.txt",
+        )
+
+    @classmethod
+    def _repository_variants(cls, config: MkosiConfig, repo: str) -> Generator[Repo, None, None]:
```

Comment:
Ditto. Also let's just name this `repository_variants()` so its named the same as the CentOS method so it gets overridden